sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -86,6 +86,9 @@ export interface ChatMessage {
     deletedOn?: Date;
     editedOn?: Date;
     id: string;
+    properties?: {
+        [propertyName: string]: string;
```

Comment:
consider `Record<string, string>` because indexer decls can be arcane to devs unfamiliar with them

Reply:
@xirzec Updated the new version